quinn-proto/src/tests/util.rs

```diff
@@ -51,6 +51,25 @@ impl Pair {
         }
     }
 
+    pub fn new_from_endpoint(client: Endpoint, server: Endpoint) -> Self {
```

Comment:
Are there any other tests that could use this?

Reply:
The reason for this change is to use separate configs to initialize client/server endpoints. Sometimes, when both client and server share the same config, the tracing log is pretty messy and becomes difficult for diagnosis. 

Current tests can switch to use this function but since they are all working fine, i don't see the motivation to make such change.

With that said, i will improve the code reuse as there are a lot of overlaps with `fn new`